modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/ClientExtensionProjectConfigurator.java

```diff
@@ -1198,6 +1198,10 @@ private void _validateClientExtension(
 			_validateRequiredTypeSettingsKeys(
 				clientExtension, "oAuthApplicationHeadlessServer");
 		}
+		else if (Objects.equals(clientExtension.type, "globalCSS")) {
+			_validateTypeSettingsValues(
+				clientExtension, "scope", "company", "layout");
```

Comment:
Question: Do you think its valuable to refactor `_validateTypeSettingsValues` ( in a later moment ) to receive a explicit List instead of a varargs param? By quickly reading in a first time, its hard to realize that "scope" is actually one param and "company" and "layout" are others :thinking: 

Does it make sense?

Reply:
It makes sense. And I plan to do it when working on refactoring frontend client extension attributes values. Currently we do not have enums or anything to help validate attributes.